Kitodo/src/main/java/org/kitodo/production/forms/LdapServerForm.java

```diff
@@ -121,6 +131,27 @@ public void setLdapServerById(int ldapServerID) {
         }
     }
 
+    /**
+     * Gets decrypted manager password.
+     *
+     * @return The decrypted manager password
+     */
+    public String getDecryptedManagerPassword() {
+        if (AESUtil.isEncrypted(ldapServer.getManagerPassword())) {
+            String securitySecret = ConfigCore.getParameterOrDefaultValue(ParameterCore.SECURITY_SECRET);
+
+            if (StringUtils.isNotBlank(securitySecret)) {
+                try {
+                    ldapServer.setManagerPassword(AESUtil.decrypt(ldapServer.getManagerPassword(), securitySecret));
+                } catch (Exception e) {
```

Comment:
Comment like above: catching every exception is not needed - catch only this exception which you will handle here.